pkg/controller/testutil/test_utils.go

```diff
@@ -46,6 +49,10 @@ import (
 	"github.com/golang/glog"
 )
 
+var (
+	keyFunc = cache.DeletionHandlingMetaNamespaceKeyFunc
```

Comment:
Just use the `keyFunc` defined in controller_utils.go

Reply:
that causes cyclical imports